src/lib/transaction_logic/mina_transaction_logic.ml

```diff
@@ -1658,7 +1683,7 @@ module Make (L : Ledger_intf.S) :
     let initial_state :
         Inputs.Global_state.t * _ Zkapp_command_logic.Local_state.t =
       ( { protocol_state = state_view
-        ; first_pass_ledger = ledger
+        ; first_pass_ledger = L.empty ~depth:0 ()
```

Comment:
I don't think you should be changing this

Reply:
Doesn't matter since you've disabled first pass ledger, but I've removed it